app/forms/fields/max_text_area_field.py

```diff
@@ -2,6 +2,7 @@
 
 
 class MaxTextAreaField(TextAreaField):
-    def __init__(self, label="", validators=None, maxlength=10000, **kwargs):
+    def __init__(self, label="", validators=None, rows=None, maxlength=10000, **kwargs):
```

Comment:
As maxlength is defaulted in `TextAreaHandler` we should remove the default here so it's consistent with `rows`.